<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapPropertiesDialog</class>
 <widget class="QWidget" name="MapPropertiesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YaRXP - Map Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tileset:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Width x Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_x">
         <property name="minimum">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_y">
         <property name="minimum">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Map ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="line_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="combo_tileset"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>BGM</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QCheckBox" name="check_auto_change_bgm">
              <property name="text">
               <string>Auto-Change BGM</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="AudioWidget" name="widget_bgm" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>BGS</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QCheckBox" name="check_auto_change_bgs">
              <property name="text">
               <string>Auto-Change BGS</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="AudioWidget" name="widget_bgs" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="button_ok">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AudioWidget</class>
   <extends>QWidget</extends>
   <header>audiowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>line_id</tabstop>
  <tabstop>line_name</tabstop>
  <tabstop>combo_tileset</tabstop>
  <tabstop>spin_x</tabstop>
  <tabstop>spin_y</tabstop>
  <tabstop>check_auto_change_bgm</tabstop>
  <tabstop>check_auto_change_bgs</tabstop>
  <tabstop>button_ok</tabstop>
  <tabstop>button_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
